Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -369,10 +369,10 @@ Ref<Inspector::Protocol::Automation::BrowsingContext> WebAutomationSession::buil
 
 // Platform-independent Commands.
 
-void WebAutomationSession::getNextContext(Ref<WebAutomationSession>&& protectedThis, Vector<Ref<WebPageProxy>>&& pages, Ref<JSON::ArrayOf<Inspector::Protocol::Automation::BrowsingContext>> contexts, Ref<WebAutomationSession::GetBrowsingContextsCallback>&& callback)
+void WebAutomationSession::getNextContext(Ref<WebAutomationSession>&& protectedThis, Vector<Ref<WebPageProxy>>&& pages, Ref<JSON::ArrayOf<Inspector::Protocol::Automation::BrowsingContext>> contexts, CommandCallback<Ref<JSON::ArrayOf<Protocol::Automation::BrowsingContext>>>&& callback)
 {
     if (pages.isEmpty()) {
-        callback->sendSuccess(WTFMove(contexts));
+        callback({ WTFMove(contexts) });
```

Comment:
are the `{` `}` actually necessary?

ditto for all the other places that just have a single value in the command result

Reply:
Good point, maybe not needed anymore.

I waffled on using std::tuple for single-argument return values, but went against it due to the ambiguity of something like `{ { "string" } }` and `{ "string" }` when the "unexpected" type is the same type as the expected type.